packages/renderer/src/lib/dashboard/NotificationCardItem.spec.ts

```diff
@@ -56,3 +57,69 @@ test('Expect notification card to show notification title, description and close
 
   expect(removeNotificationMock).toBeCalled();
 });
+
+test('Test info notification style and icon', () => {
+  const notification: NotificationCard = {
+    id: 1,
+    extensionId: 'extension',
+    title: 'Info notification title',
+    body: 'Info notification description',
+    type: 'info',
+  };
+  render(NotificationCardItem, {
+    notification,
+  });
+  const icon = screen.getAllByRole('img', { hidden: true })[0];
```

Comment:
Using getAllByRole and taking the first element can be a bit fragile, especially since the order of icons might change or more icons could be added later.

Maybe we could use this solution (and merge it in another PR):
https://stackoverflow.com/questions/62587372/query-font-awesome-element-in-react-testing-library

Reply:
I'll create a new PR for this after this